src/excmds.ts

```diff
@@ -296,6 +296,44 @@ export function cssparse(...css: string[]) {
     console.log(CSS.parse(css.join(" ")))
 }
 
+/** @hidden */
+//#background
+export async function loadtheme(themename: string) {
+    if (!(await Native.nativegate("0.1.8")))
+        return
+    const separator = (await browserBg.runtime.getPlatformInfo().os) == "win" ? "\\" : "/"
+    // remove the "tridactylrc" bit so that we're left with the directory
+    const path = (await Native.getrcpath()).split(separator).slice(0, -1).join(separator)
+        + separator +  "themes" + separator + themename
+    const file = await Native.read(path)
+    if (file.code != 0)
+        throw new Error("Couldn't read theme " + path)
+    return set("customthemes." + themename, file.content)
+}
+
+/** @hidden */
+//#background
+export async function unloadtheme(themename: string) {
+    unset("customthemes." + themename)
+}
+
+/** Changes the current theme.
+ * 
+ * If THEMENAME is "dark", "default", "greenamt", "quake" or "shydactyl", the theme will be loaded from Tridactyl's internal storage.
+ *
+ * If THEMENAME is set to any other value, Tridactyl will attempt to use its native binary (see [[native]]) in order to load a CSS file named THEMENAME from disk. The CSS file has to be in a directory named "themes" and this directory has to be in the same directory as your tridactylrc.
+ *
+ * Example: `:colourscheme mysupertheme.css`
+ */
+//#background
+export async function colourscheme(themename: string) {
+    // If this is a builtin theme, no need to bother with native messaging stuff
+    if (["dark", "default", "greenmat", "quake", "shydactyl"].includes(themename))
```

Comment:
We should have one place in the code where the default themes are listed and import it. Ideally, that list would be generated from looking at the themes folder.

Having them hard-coded in the excmds doesn't make much sense to me.